<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\Annemartijn\Documents\GitHub\nhibernate-core\NHibernate.nunit" total="1" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2014-04-10" time="15:01:05">
  <environment nunit-version="2.6.3.13283" clr-version="2.0.50727.5477" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\Users\Annemartijn\Documents\GitHub\nhibernate-core" machine-name="ILIANLAPTOPWERK" user="Annemartijn" user-domain="IlianLaptopWerk" />
  <culture-info current-culture="nl-NL" current-uiculture="en-US" />
  <test-suite type="Test Project" name="C:\Users\Annemartijn\Documents\GitHub\nhibernate-core\NHibernate.nunit" executed="True" result="Failure" success="False" time="1.673" asserts="0">
    <results>
      <test-suite type="Assembly" name="C:\Users\Annemartijn\Documents\GitHub\nhibernate-core\src\NHibernate.Test\bin\Debug-2.0\NHibernate.Test.dll" executed="True" result="Failure" success="False" time="0.145" asserts="0">
        <results>
          <test-suite type="Namespace" name="NHibernate" executed="True" result="Failure" success="False" time="0.143" asserts="0">
            <results>
              <test-suite type="Namespace" name="Test" executed="True" result="Failure" success="False" time="0.142" asserts="0">
                <results>
                  <test-suite type="Namespace" name="Criteria" executed="True" result="Failure" success="False" time="0.137" asserts="0">
                    <results>
                      <test-suite type="Namespace" name="Lambda" executed="True" result="Failure" success="False" time="0.137" asserts="0">
                        <results>
                          <test-suite type="TestFixture" name="IntegrationFixtureAsync" executed="True" result="Failure" success="False" time="0.136" asserts="0">
                            <results>
                              <test-case name="NHibernate.Test.Criteria.Lambda.IntegrationFixtureAsync.OnClauseAsync" executed="True" result="Error" success="False" time="0.052" asserts="0">
                                <failure>
                                  <message><![CDATA[System.AggregateException : One or more errors occurred.
  ----> NUnit.Framework.AssertionException : 0 Should Be Equal To 1.]]></message>
                                  <stack-trace><![CDATA[at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
at System.Threading.Tasks.Task.Wait()
at NHibernate.Test.Criteria.Lambda.IntegrationFixtureAsync.OnClauseAsync() in c:\Users\Annemartijn\Documents\GitHub\nhibernate-core\src\NHibernate.Test\Criteria\Lambda\IntegrationFixtureAsync.cs:line 178
--AssertionException
at NHibernate.Test.Criteria.Lambda.IntegrationFixtureAsync.<OnClauseAsync>b__10(Task`1 task) in c:\Users\Annemartijn\Documents\GitHub\nhibernate-core\src\NHibernate.Test\Criteria\Lambda\IntegrationFixtureAsync.cs:line 187
at System.Threading.Tasks.Task.Execute()
]]></stack-trace>
                                </failure>
                              </test-case>
                            </results>
                          </test-suite>
                        </results>
                      </test-suite>
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>